mbed_host_tests/host_tests_conn_proxy/conn_primitive_serial.py

```diff
@@ -50,20 +49,26 @@ def __init__(self, name, port, baudrate, config):
             self.logger.prn_inf("serial port changed from '%s to '%s')"% (self.port, serial_port))
             self.port = serial_port
 
-        try:
-            # TIMEOUT: While creating Serial object timeout is delibrately passed as 0. Because blocking in Serial.read
-            # impacts thread and mutliprocess functioning in Python. Hence, instead in self.read() s delay (sleep()) is
-            # inserted to let serial buffer collect data and avoid spinning on non blocking read().
-            self.serial = Serial(self.port, baudrate=self.baudrate, timeout=0)
-        except SerialException as e:
-            self.serial = None
-            self.LAST_ERROR = "connection lost, serial.Serial(%s, %d, %d): %s"% (self.port,
-                self.baudrate,
-                self.timeout,
-                str(e))
-            self.logger.prn_err(str(e))
-        else:
-            self.reset_dev_via_serial(delay=self.forced_reset_timeout)
+        startTime = time.time()
+        while time.time() - startTime < self.serial_pooling:
+            self.logger.prn_inf("serial(port=%s, baudrate=%d, timeout=%s)"% (self.port, self.baudrate, self.timeout))
```

Comment:
Assuming this is a print , do we really want to do this every time around the while ?